sdk/tests/types/address/alias.rs

```diff
@@ -87,18 +87,15 @@ fn debug() {
 fn bech32() {
     let address = Address::from(AliasAddress::from_str(ALIAS_ID).unwrap());
 
-    assert_eq!(address.to_bech32("rms"), ALIAS_BECH32);
+    assert_eq!(address.to_bech32("rms").to_string(), ALIAS_BECH32);
```

Comment:
In case you want to remove the to_string since you removed it for ed25519 and nft. Otherwise just resolve.